src/Saritasa.Tools.Messages/Common/Expressions/ExpressionExecutor.cs

```diff
@@ -79,7 +79,7 @@ public dynamic Execute(MethodInfo info, params dynamic[] parameters)
                 return func.Invoke(parameters[0], parameters[1], parameters[2], parameters[3]);
```

Comment:
[TODO] 
Use dictionary for more compact execution and pass more than 8 parameters.